### PR TITLE
[hi] Localized content/en/docs/reference/glossary/resource-quotas.md in hindi

### PR DESCRIPTION
**Pull request by KashishLakhara04**
fixes: #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/resource-quota.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: संसाधन कोटा (Resource Quotas)
+id: resource-quota
+date: 2018-04-12
+full_link: /docs/concepts/policy/resource-quotas/
+short_description: >
+  प्रति नामस्थान पर कुल संसाधन खपत को सीमित करने वाली बाधाएं प्रदान करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/resource-quota.md:7]:
```suggestion
  प्रति नेमस्पेस पर कुल संसाधन खपत को सीमित करने वाली बाधाएं (constraints) प्रदान करता है।
```

File: content/hi/docs/reference/glossary/resource-quota.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+aka: 
+tags:
+- fundamental
+- operation
+- architecture
+---
+ प्रति {{< glossary_tooltip text="नामस्थान" term_id="namespace" >}} पर कुल संसाधन खपत को सीमित करने वाली बाधाएं प्रदान करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/resource-quota.md:15]:
```suggestion
 प्रति {{< glossary_tooltip text="नेमस्पेस" term_id="namespace" >}} पर कुल संसाधन खपत को सीमित करने वाली बाधाएं (contraints) प्रदान करता है।
```

File: content/hi/docs/reference/glossary/resource-quota.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+<!--more--> 
+
+किसी नामस्थान में प्रकार द्वारा बनाई जा सकने वाली वस्तुओं की मात्रा को सीमित करता है, साथ ही उस परियोजना में संसाधनों द्वारा उपभोग किए जा सकने वाले कंप्यूट संसाधनों की कुल मात्रा को भी सीमित करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/resource-quota.md:19]:
```suggestion
किसी नेमस्पेस में बनाई जा सकने वाली ऑब्जेक्ट्स की मात्रा को उनके प्रकार के अनुसार सीमित करता है, साथ ही उस परियोजना के संसाधनों द्वारा उपभोग किए जा सकने वाले कंप्यूट संसाधनों की कुल मात्रा को भी सीमित करता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/resource-quota.md (hunk 3 above)
Comment (KashishLakhara04) [reply content/hi/docs/reference/glossary/resource-quota.md]: @anubha-v-ardhan  all the suggested changes have been applied

**Review by anubha-v-ardhan — COMMENTED** [review]
/lgtm
/approve

Thank you @KashishLakhara04!

**Pull request merged**